bika/lims/browser/clientfolder.py

```diff
@@ -168,6 +171,11 @@ def folderitem(self, obj, item, index):
         :return: the dict representation of the item
         :rtype: dict
         """
+        # Call the folderitem method from the base class
+        item = BikaListingView.folderitem(self, obj, item, index)
+        if not item:
+            return None
```

Comment:
Please also use the `super` call here instead of calling the named base class, e.g.:
https://github.com/senaite/senaite.core/blob/master/bika/lims/browser/worksheet/views/analyses_transposed.py#L70

Btw, when can this happen that the `item` is `None`?
The thing with `None` values here is that the listing count would be wrong and I'm not sure if this is handled gracefully in `senaite.core.listing`, because it expects there dicts as folderitems

Reply:
> Btw, when can this happen that the item is None?

Indeed, it should not happen. And if it somehow happens, better to handle that as bug than obfuscating the anomaly here. So I agree with you